src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/InputJavadocContentLocationDefault.java

```diff
@@ -0,0 +1,72 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadoccontentlocation;
+
+/**
+ * config: default
+ * location="SECOND_LINE"
+ */
+public interface InputJavadocContentLocationDefault {
```

Comment:
I suggest to add test cases for javadoc before class and before package, because they are put inside MODIFIERS token by parser and there can problems with them.

Reply:
done. Actually, this check handles the `BLOCK_COMMENT_BEGIN` tag itself. It doesn't matter where it is located.